### DISCUSSION TITLE
How to change the color of plus (+) sign of a new window button?

### DISCUSSION THREAD

**Discussion by azzamsa** [wezterm/wezterm#1183] [Q&A] [answered]
Hi.

I have tried so many times to change the style of this button, but no luck:

![2021-09-28_18-01](https://user-images.githubusercontent.com/17734314/135078426-6f0b7c7f-6702-49ac-8d00-5e189a890965.png)

I tried to change the color in `ansi`, `brights`, and other parts. 
However, I can modify other parts as I wish.

**Comment by wez** [comment, marked as answer]
Take a look at: https://wezfurlong.org/wezterm/config/appearance.html#tab-bar-appearance--colors

```lua
return {
  colors = {
    tab_bar = {
      new_tab = {
        bg_color = "red",
        fg_color = "yellow",
      }
    }
  }
}
```

**Reply by azzamsa** [reply]
Thank you!
